mypy.ini

```diff
@@ -0,0 +1,2 @@
+[mypy-mistletoe]
```

Comment:
Could you consider renaming this file to `.mypi.ini` to make it hidden (most of the config files in the base directory are hidden)?

Reply:
If the name is `.mypy.ini`, then will mypy call from repo catch the file correctly? I saw mypy doc mention that it reads `mypy.ini` in the repo top automatically. If we make it hidden, we always need `--config` option to call mypy.